test/buildd/test_reflect.py

```diff
@@ -31,7 +31,7 @@ class _BaseBuildReflectTest(
     mixins.PrototypedDirMixin,
     unittest.TestCase,
 ):
-    proto_dir = "build/bf-projects/bf_selfbuild_project"
+    proto_dir = "buildd/bf-projects/bf_selfbuild_project"
```

Comment:
why buildd instead of build?

Reply:
'pytest' ignores 'build' and 'dist' directories by default. https://github.com/pytest-dev/pytest/issues/1544
It might be "fixed" by redefining configuration for pytest in pyproject.toml OR by renaming directories.